core/iwasm/libraries/wasi-nn/README.md

```diff
@@ -0,0 +1,43 @@
+# WASI-NN
+
+## How to use
+
+Enable WASI-NN in the WAMR by spefiying it in the cmake building configuration as follows,
+
+```
+set (WAMR_BUILD_WASI_NN  1)
+```
+
+The definition of the functions provided by WASI-NN is in the header file `core/iwasm/libraries/wasi-nn/wasi_nn.h`.
+
+By only including this file in your WASM application you will bind WASI-NN into your module.
+
+## Tests
+
+To run the tests we assume that the current directory is the root of the repository.
+
+
+1. Build the dockre image,
+
+```
+docker build -t wasi-nn -f core/iwasm/libraries/wasi-nn/test/Dockerfile .
```

Comment:
Not sure whether the test is just a simple interface test (like unit test), or a sample? If it is a sample, please put it under `<wamr_root>/samples`, for example, samples/wasi-nn.

Reply:
I'm testing multiple neural network models. Once it's ready for a release we can add a real sample!